src/main/resources/com/google/api/codegen/go/example.snip

```diff
@@ -6,7 +6,11 @@
     package {@view.fileHeader.localExamplePackageName}
 
     import (
-        {@imports(view.fileHeader.imports)}
+        @if view.fileHeader.importSection.standardImports
+            {@imports(view.fileHeader.importSection.standardImports)}
+
+        @end
+        {@imports(view.fileHeader.importSection.externalImports)}
```

Comment:
These five lines are repeated 4 times for go - maybe consolidate into one snippet function?